public/app/plugins/datasource/azuremonitor/azure_log_analytics/azure_log_analytics_datasource.test.ts

```diff
@@ -42,13 +48,13 @@ describe('AzureLogAnalyticsDatasource', () => {
     it('should return a schema to use with monaco-kusto', async () => {
       const result = await ctx.datasource.azureLogAnalyticsDatasource.getKustoSchema('myWorkspace');
```

Comment:
Great catch! This will fix the type assertion. We still need the `?` as `database` can still be undefined but anything that removes a type assertion is a nice change 😊